### PR TITLE
Setup github action to run tests

### PR DESCRIPTION
**Pull request by leeallen337**
## Why?

Setup GitHub actions to run tests and deprecate wercker

## What changed?
- Setup GitHub actions to run tests
- Use the official [`checkout`](https://github.com/marketplace/actions/checkout) action
- Use the official [`setup-node`](https://github.com/marketplace/actions/setup-node-js-environment) action

### What's next?
- Automatically publishing a release
  - https://help.github.com/en/actions/language-and-framework-guides/publishing-nodejs-packages#publishing-packages-to-the-npm-registry
  - Thoughts?
    - We probably don't want to automatically do it on a merge to master as we could merge things to master that don't require a release

- [ ] Did you update the CHANGELOG?
  - No changelog update needed

### CODE REVIEW

**Comment by leeallen337** [comment]
Looks like wercker is failing obviously because I deleted the wercker file. I removed the wercker webhook from this repository's settings. Anyone know anything else I should do?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    branches-ignore:
+      - 'master'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
```
Comment (branchvincent) [inline .github/workflows/test.yml:13]: fyi looks like that's the [default](https://github.com/actions/checkout/blob/master/action.yml#L58)

Comment (leeallen337) [reply .github/workflows/test.yml:13]: Nice. I may just leave it in there to be explicit

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# contxt-sdk [![wercker status](https://app.wercker.com/status/869ef086297da79ddd0cbf3564f7cba6/s/master 'wercker status')](https://app.wercker.com/project/byKey/869ef086297da79ddd0cbf3564f7cba6)
+# contxt-sdk
```
Comment (branchvincent) [inline README.md:1]:
```suggestion
# contxt-sdk ![Test](https://github.com/ndustrialio/contxt-sdk-js/workflows/Test/badge.svg)
```

Comment (leeallen337) [reply README.md:1]: Thanks. I was trying to find the badges but I couldn't

Comment (branchvincent) [reply README.md]: Took me a while to find it too: it's [here](https://github.com/ndustrialio/contxt-sdk-js/actions?query=workflow%3ATest) (top right corner, `Create status badge`)

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '8.x'
```
Comment (greg5green) [inline .github/workflows/tests.yml:16]: probably could go higher than 8 -- think 8 was just the highest LTS at the time the wercker file was set up?

Comment (leeallen337) [reply .github/workflows/tests.yml:16]: I'll work on upgrading node. I tried and it seemed like some things broke so I'll put it in another pull request 👍

**Pull request merged**